pkg/utils/kubernetes.go

```diff
@@ -158,7 +172,35 @@ func DeleteOldCStorVolumeAttachmentCRs(volumeID string) error {
 		if err != nil {
 			return err
 		}
+
+		// Extract only the CVA which belongs to the current node
+		if strings.Contains(csivol.Name, nodeID) {
```

Comment:
Instead of checking the entire string can we check the last part?
```go
if csivol.Name[len(csivol.Name)-len(nodeID):] == nodeID  { //.. }
                                (Or)
if strings.HasSuffix(csivol.Name, nodeID) { //.. }
```